### PR TITLE
Mirid updates for v1.3.4

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: docfx/api-override/Meadow.Foundation.mikroBUS/Meadow.Foundation.mikroBUS.Sensors.CLEM.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
### Code Example
+
+```csharp
+CLEM currenClick;
+
+public MeadowApp()
+{
+    Console.WriteLine("Initializing ...");
+}
+
```
Comment (patridge) [inline docfx/api-override/Meadow.Foundation.mikroBUS/Meadow.Foundation.mikroBUS.Sensors.CLEM.md:20]: Not remotely critical, but can we trim all the trailing line breaks before we close a code block? (Not just this file, but all, I think.)

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Mcp3008.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+    IDigitalOutputPort chipSelectPort = Device.CreateDigitalOutputPort(Device.Pins.D01);
+
+    mcp = new Mcp3008(Device.CreateSpiBus(), chipSelectPort);
+
+    port = mcp.CreateAnalogInputPort(mcp.Pins.CH0, 32, TimeSpan.FromSeconds(1), new Voltage(3.3, Voltage.UnitType.Volts), Mcp3xxx.InputType.SingleEnded);
+
+    port.Updated += (s, result) =>
+    {
+        Console.WriteLine($"Analog event, new voltage: {result.New.Volts:N2}V, old: {result.Old?.Volts:N2}V");
```
Comment (patridge) [inline docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Mcp3008.md:31]: Probably should be a Resolver Log

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Pca8574.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+            device.AllOff();
+
+            Thread.Sleep(1000);
+
+            // turn them all on
+            device.AllOn();
+            Thread.Sleep(1000);
+
+            // turn on just one
+            Console.WriteLine($"Update pin {i} to {true}");
```
Comment (patridge) [inline docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Pca8574.md:70]: Same for Resolver.Log

**Review by patridge — APPROVED** [review]
LGTM. We seem to have some Console.WriteLine calls in the source code pulled in here. Nothing to stop us from merging, though.

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Mcp3008.md (hunk 2 above)
Comment (adrianstevens) [reply docfx/api-override/Meadow.Foundation/Meadow.Foundation.ICs.IOExpanders.Mcp3008.md:31]: that's a Meadow.Foundation issue vs docs - but agreed

**Pull request merged**